src/lib/core/gestures/MdGestureConfig.ts

```diff
@@ -4,16 +4,25 @@ import {HammerGestureConfig} from '@angular/platform-browser';
 /* Adjusts configuration of our gesture library, Hammer. */
 @Injectable()
 export class MdGestureConfig extends HammerGestureConfig {
+  private _hammer = typeof window !== 'undefined' ? (window as any).Hammer : null;
```

Comment:
I went with `any` here and a bit below, because it throws a TS compile error if HammerJS is missing.